### PR TITLE
Don't track help indexes in git

### PR DESCRIPTION
**Pull request by edk0**
Remembering to check in regenerated ones every time we change the help is pointless extra work and creates pointless extra merge conflicts if there are two PRs with new help files in flight

No doubt this will conflict with #23, demonstrating the problem somewhat.

### CODE REVIEW

File: help/Makefile.am [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Generated automatically from Makefile.in by configure.
```
Comment (glguy) [inline help/Makefile.am:1]: Is this true? If so why are we modifying it? If not can we delete it?

File: help/Makefile.am [hunk 2]
```diff
@@ -17,10 +17,12 @@ SYMLINKS=	topic accept cmode admin names links away whowas \
 
 all: index
 build:
-clean:
 depend:
 lint:
 
+clean:
```
Comment (glguy) [inline help/Makefile.am:23]: We're adding a new `PHONY` target and there are a bunch of other PHONY targets. Normally I'd expect

```
.PHONY: all build clean depend lint index
```

Opinion?

**Review by glguy — CHANGES_REQUESTED** [review]
If happy to be wrong on either of these changes.

File: help/Makefile.am (hunk 1 above)
Comment (edk0) [reply help/Makefile.am:1]: I think it's there so it shows up in the generated Makefile. The next line is definitely wrong, though.

File: help/Makefile.am (hunk 2 above)
Comment (edk0) [reply help/Makefile.am:23]: We're not adding it, just moving it down a few lines. I can't remember how but some of the autostuff generates a .PHONY in the Makefile proper.

**Review by glguy — DISMISSED** [review]

**Review by glguy — APPROVED** [review]

**Pull request merged**